tests/fast-integration/monitoring-test/.env.template

```diff
@@ -0,0 +1,7 @@
+KEB_HOST=
```

Comment:
Do you need it? It was used to provision Kyma with KEB. I don't see you do it.

Reply:
Removed